tests/e2e/test-data/expected-html/dist.txt

```diff
@@ -1 +1,8 @@
-<!DOCTYPE html><html><head><title>Index of /dist/</title><meta name='viewport' content='width=device-width, initial-scale=1.0' /><meta charset='utf-8' /><style type='text/css'>td { padding-right: 16px; text-align: right; font-family: monospace }td:nth-of-type(1) { text-align: left; overflow-wrap: anywhere }td:nth-of-type(3) { white-space: nowrap } th { text-align: left; } @media(prefers-color-scheme: dark) { body { color: white; background-color:#1c1b22; } a { color: #3391ff; } a:visited { color: #C63B65; } }</style></head><body><h1>Index of /dist/</h1><table><tr><th>Filename</th><th>Modified</th><th>Size</th></tr><tr><td><a href='../'>../</a></td><td>-</td><td>-</td></tr><tr><td><a href='/dist/latest/'>latest/</a></td><td>-</td><td>-</td></tr><tr><td><a href='/dist/index.json'>index.json</a></td><td>2023-09-12 05:43Z</td><td>18 B</td></tr></table></body></html>
\ No newline at end of file
+<!DOCTYPE html><html>
+<head><title>Index of /dist/</title></head>
+<body>
+<h1>Index of /dist/</h1><hr><pre><a href="../">../</a><br />
+<a href="latest/">latest/</a> - -<br />
+<a href="/dist/index.json">index.json</a> 2023-09-12 05:43Z 18 B<br />
```

Comment:
There's no `Z` at the end of the time in the nginx output.

Reply:
Yeah we don't need TZ info, let's always make it UTC.